File: app/src/main/java/tech/leonam/resultadodocertame/model/entidade/TurmaEntidade.java
package tech.leonam.resultadodocertame.model.entidade;

import java.util.List;

public class TurmaEntidade {
    private String nomeDaTurma;
    private List<AlunoEntidade> turma;

    public TurmaEntidade(List<AlunoEntidade> turma,String nomeDaTurma) {
        this.turma = turma;
        this.nomeDaTurma = nomeDaTurma;
    }

    public TurmaEntidade() {
    }

    public List<AlunoEntidade> getTurma() {
        return turma;
    }

    public void setTurma(List<AlunoEntidade> turma) {
        this.turma = turma;
    }

    public String getNomeDaTurma() {
        return nomeDaTurma;
    }

    public void setNomeDaTurma(String nomeDaTurma) {
        this.nomeDaTurma = nomeDaTurma;
    }
}

File: app/src/main/java/tech/leonam/resultadodocertame/model/entidade/TurmaEntidade.kt
package tech.leonam.resultadodocertame.model.entidade

class TurmaEntidade {
    var nomeDaTurma: String? = null
    var turma: List<AlunoEntidade>? = null

    constructor()

    constructor(turma: List<AlunoEntidade>?, nomeDaTurma: String?) {
        this.turma = turma
        this.nomeDaTurma = nomeDaTurma
    }
}